testrunner/docker/Dockerfile

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018 Google LLC. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM launcher.gcr.io/google/debian9
+
+COPY testrunner /bin/testrunner
+
+RUN apt-get update && \
+	    apt-get install -y --no-install-recommends wget && \
+	    rm -rf /var/lib/apt/lists/*
+
+COPY oss.sha512.checksums /oss.sha512.checksums
+
+RUN set -xeu && \
+      mkdir -p /usr/share/testrunner && \
+      wget -q -O /usr/share/testrunner/testrunner.LICENSE https://raw.githubusercontent.com/GoogleCloudPlatform/marketplace-k8s-app-tools/master/LICENSE && \
```

Comment:
A few more thoughts here:
- We also need to distribute the source code in addition to the license files.
- I think we need a way to automate that for go packages. We want to copy the source that was used to build the go binary. Thus, I think we will want to vendor in these libraries to make sure they are stable and consistent.
- Another thought is the Dockerfile should compile and build the binary too, instead of relying on an external build step.

All of these are much larger scope for this change. It has to do with go tooling. I've filde an issue on the new repo and will follow up separately: https://github.com/GoogleCloudPlatform/marketplace-testrunner/issues/2